File: src/main/java/lesson/java/basics/Variables.java
package lesson.java.basics;

public class Variables {
    public static void main(String args[]){
        final String str1 = "hi";
        String str2 = "he";
        str2 += "llo";
        final int int1 = 100;
        int int2 = 200;
        int2 += 100;
        System.out.println(
            str1 + str2 + int1 + int2);
    }
}

File: src/main/scala/lesson/scala/basics/Variables.scala
package lesson.scala.basics

object Variables {
  def main(args: Array[String]) = {
    val str1 = "hi"
    var str2 : String = "he"
    str2 += "llo"
    val int1 : Int= 100
    var int2 = 200
    int2 += 100
    println(str1 + str2 + int1 + int2)
  }
}
